api/src/interpreter/entities/interpreter.entity.ts

```diff
@@ -155,6 +155,7 @@ export class InterpreterEntity {
       'gst',
       'comments',
       'contractExtension',
+      'criminal'
```

Comment:
Do we have `criminal` column? I only see `criminalRecordCheck` and `criminalRecordCheckDate`

Reply:
FIXED